src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -55,6 +59,9 @@
  */
 bool AddAllLocalTablesToMetadata = true;
 
+static SPIPlanPtr plan_getviewrule = NULL;
```

Comment:
@agedemenli is this PR relevant anymore?

Reply:
> @agedemenli is this PR relevant anymore?

@agedemenli ??

Reply:
nope, closing it